src/Counters.jl

```diff
@@ -1,8 +1,12 @@
 using Base.Iterators: flatten
+using Distributions: Normal, truncated
+using Random: rand
 
-flops(a, b, size) = prod(ind -> size[ind], a ∪ b, init=1)
-flops(a, b, size, keep) = prod(ind -> size[ind], flatten((a ∪ b, ∩(keep, a, b))), init=1)
+flops(a, b, size) = prod(ind -> size[ind], a ∪ b, init=one(BigInt))
+flops(a, b, size, keep) = prod(ind -> size[ind], flatten((a ∪ b, ∩(keep, a, b))), init=one(BigInt))
 
 rank(a, b, size, keep) = length(symdiff(a, b) ∪ ∩(keep, a, b))
 
-removedsize(a, b, size, keep) = prod(size[ind] for ind ∈ symdiff(a, b) ∪ (keep ∩ a) ∪ (keep ∩ b); init=1) - prod(size[ind] for ind ∈ a) - prod(size[ind] for ind ∈ b)
\ No newline at end of file
+removedsize(a, b, size, keep) = prod(size[ind] for ind ∈ symdiff(a, b) ∪ (keep ∩ a) ∪ (keep ∩ b); init=1) - prod(size[ind] for ind ∈ a) - prod(size[ind] for ind ∈ b)
+
+removedsize_noise(a, b, size, keep) = rand(truncated(Normal(1.0, 0.01),0,10),1)[1] * (prod(size[ind] for ind ∈ symdiff(a, b) ∪ (keep ∩ a) ∪ (keep ∩ b); init=1) - prod(size[ind] for ind ∈ a) - prod(size[ind] for ind ∈ b))
```

Comment:
Is it not just easier (and cleaner) to just call `removedsize` and multiply by the normal noise in the `RandomGreedy` solver?

Reply:
Being capable of choosing the distribution would make more sense. Also, I would not create a new function for this because we might want to use other heuristics.

```suggestion
removedsize(a, b, size, keep) = prod(size[ind] for ind ∈ symdiff(a, b) ∪ (keep ∩ a) ∪ (keep ∩ b); init=1) - prod(size[ind] for ind ∈ a) - prod(size[ind] for ind ∈ b)
```